_projects/2022/100385586/100385586.Rmd

```diff
@@ -0,0 +1,310 @@
+---
+title: "Final project Alejandro Aísa" 
```

Comment:
Please use a more descriptive title (about the data and the chart). ;-) See examples: https://csslab.uc3m.es/dataviz/projects.html